Compatibility/runtimeconfig.template.json

```diff
@@ -0,0 +1,170 @@
+
+    // Switch to allow changes during a VisualTreeChanged event
+    "Switch.System.Windows.Diagnostics.AllowChangesDuringVisualTreeChanged": false,
+
+    // Switch to disable automatic touch keyboard invocation on focus of a control
+    "Switch.System.Windows.Input.Stylus.DisableImplicitTouchKeyboardInvocation": false,
+
+    // Desktop/Interactive Window Stations:
+    //  Rendering will be throttled back/stopped when no display devices are available. For e.g., when a TS 
+    //  session is in WTSDisconnected state, the OS may not provide any display devices in response to our enumeration.
+    //
+    //  If an application would like to continue rendering in the absence of display devices (accepting that 
+    //  it can lead to a CPU spike), it can set 
+    //  to true.
+    //
+    // Service/Non-interactive Window Stations
+    //  Rendering will continue by default, irrespective of the presence of display devices.Unless the WPF
+    //  API's being used are short-lived (like rendering to a bitmap), it can lead to a CPU spike. 
+    //  If an application running inside a service would like to receive the 'default' WPF behavior, 
+    //  i.e., no rendering in the absence of display devices, then it should set
+    //  to true
+    //
+    // In pseudo-code, 
+    //  IsNonInteractiveWindowStation = !Environment.UserInteractive
+    //    IF DisplayDevicesNotFound() THEN
+    //      IF IsNonInteractiveWindowStation THEN 
+    //        // We are inside a SCM service
+    //        // Default = True, AppContext switch can override it to False
+    //        ShouldRender = !CoreAppContextSwitches.ShouldNotRenderInNonInteractiveWindowStation
+    //      ELSE 
+    //        // Desktop/interactive mode, including WTSDisconnected scenarios
+    //        // Default = False, AppContext switch can override it to True
+    //        ShouldRender = CoreAppContextSwitches.ShouldRenderEvenWhenNoDisplayDevicesAreAvailable
+    //        END IF
+    //      END IF"
+    "Switch.System.Windows.Media.ShouldRenderEvenWhenNoDisplayDevicesAreAvailable": false,
+    "Switch.System.Windows.Media.ShouldNotRenderInNonInteractiveWindowStation": false,
+
+    // Reserved for future use
+    "Switch.System.Windows.DoNotUsePresentationDpiCapabilityTier3OrGreater": false,
+
+    // 
+    "Switch.System.Windows.AllowExternalProcessToBlockAccessToTemporaryFiles": false,
+
+    // Malicious managed objects could be placed in the clipboard lying about its format, 
+    // to fix this OleConverter now restricts object deserialization in some cases.
+    // When this switch is enabled behavior falls back to deserializing without restriction.
+    "Switch.System.Windows.EnableLegacyDangerousClipboardDeserializationMode": false,
+
+    // FrameworkAppContextSwitches
+
+    // Switch to enable non-adorner based rendering of TextSelection in TextBox and PasswordBox
+    "Switch.System.Windows.Controls.Text.UseAdornerForTextboxSelectionRendering": true,
```

Comment:
Add link to release notes for this: https://github.com/microsoft/dotnet/blob/master/Documentation/compatibility/wpf-TextBox-PasswordBox-text-selection-does-not-follow-system-colors.md

Reply:
Just because it's hard to understand what this means from the short comment alone.